contracts/config/networks.json

```diff
@@ -1,6 +1,6 @@
 {
     "localGeth": {
-      "url": "http://127.0.0.1:37000",
```

Comment:
Did you mean to change this port and the EncRpcUrl one?

Reply:
Yep, those are the local testnet ports, someone changed them to something else, but this should be the default state in order to run hardhat stuff locally.